src/game/level.c

```diff
@@ -27,6 +27,12 @@ struct level_t
     boxes_t *boxes;
 };
 
+// A wrapper for fclose to pass like lt_destroy_t
+static void fclose_lt(void* file)
```

Comment:
Yes, I think we need to move this adapter to a separate file. Please create a header `src/system/lt/lt_adapters.h` and move `fclose_lt` there. Let's keep it header-only for now. Also, please use include guards for the header (no `#pragma once`). Thanks!

Reply:
> Let's keep it header-only for now.

But if I include the `"lt_adapters.h"` in multiple files (namely _level.c_ and _platforms.c_) there will be multiple definitions of `fclose_lt`.

Reply:
@geekdima ok, let's created the c file then. :)